src/controller/AbstractMdnsDiscoveryController.cpp

```diff
@@ -0,0 +1,100 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2013-2017 Nest Labs, Inc.
```

Comment:
NestLabs? looks like a new file. What is its origin?

Reply:
Removed. Several other files in the SDK seem to have that header for some reason and it looks like I copied that in with the rest of the header.